pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
Will it really solve all problems ?

Cube itself doesn't have a method to write itself to disk.
`write` method is implemented only in subclasses.

It can be extended to the `Cube` class, but then what kind of file extension should we use ?